### PR TITLE
added one step to fire point in ukrainian version

### PR DESCRIPTION
**Pull request by drwebmaker**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1519)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: ukrainian/fire/fire.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 ﻿Смажений зефір чудово пахне, але чи варто його пробувати? Може він не їстівний?
-Варто спробувати, мабуть.
\ No newline at end of file
+Варто спробувати, мабуть.
+
+[Розженуся та спробую прорватися крізь одну з стін](break-through-wall/break-through-wall.md)
```
Comment (casey-collab) [inline ukrainian/fire/fire.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: ukrainian/fire/fire.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+
+[Скоштую](marshmallow.md)
```
Comment (casey-collab) [inline ukrainian/fire/fire.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**